Fix failing tests related to rounds-related SQL triggers

### Expected behaviour

Tests always pass.

### Actual behaviour

```
26 passing (1m)
13 failing

  1) Rounds-related SQL triggers
       genesisBlock
         should apply genesis block transactions to mem_accounts (native):

      AssertionError: expected 95 to equal 103
      + expected - actual

      -95
      +103
      
    at test/unit/sql/rounds.js:243:45
    at tryCatcher (node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (node_modules/bluebird/js/release/promise.js:693:18)
    at Async._drainQueue (node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.Async.drainQueues (node_modules/bluebird/js/release/async.js:17:14)
  

  2) Rounds-related SQL triggers
       round
         should forge block with 1 TRANSFER transaction to random account, update mem_accounts (native) and delegates (trigger block_insert_delete) tables:
     Error: Invalid block height
    at maybeWrapAsError (node_modules/bluebird/js/release/util.js:61:12)
    at node_modules/bluebird/js/release/nodeback.js:38:50
    at test/unit/sql/rounds.js:464:5
    at node_modules/async/dist/async.js:421:16
    at next (node_modules/async/dist/async.js:5278:29)
    at node_modules/async/dist/async.js:906:16
    at test/unit/sql/rounds.js:451:5
    at node_modules/async/dist/async.js:3838:9
    at node_modules/async/dist/async.js:421:16
    at iterateeCallback (node_modules/async/dist/async.js:924:17)
    at node_modules/async/dist/async.js:906:16
    at node_modules/async/dist/async.js:3835:13
    at Immediate.<anonymous> (test/unit/sql/rounds.js:444:25)
  

  3) Rounds-related SQL triggers
       round
         should calculate rewards for round 1 correctly - all should be the same (native, rounds_rewards, delegates):

      AssertionError: expected {} to deeply equal { Object (f25af3c59ac7f5155c7a9f36762bd941b9dc9c5c051a1bc2d4e34ed773dd04a3, 74583aba9c0b92e4f08c8c75e6df341c255ca007971195ff64d6f909dc4b7177, ...) }
      + expected - actual

      -{}
      +{
      +  "01389197bbaf1afb0acd47bbfeabb34aca80fb372a8f694a1c0716b3398db746": {
      +    "fees": 12277227
      +    "pk": "01389197bbaf1afb0acd47bbfeabb34aca80fb372a8f694a1c0716b3398db746"
      +    "rewards": 0
      +  }
      +  "0186d6cbee0c9b1a9783e7202f57fc234b1d98197ada1cc29cfbdf697a636ef1": {
      +    "fees": 12277227
      +    "pk": "0186d6cbee0c9b1a9783e7202f57fc234b1d98197ada1cc29cfbdf697a636ef1"
      +    "rewards": 0
      +  }
      +  "031e27beab583e2c94cb3167d128fc1a356c1ae88adfcfaa2334abffa3ae0b4c": {
      +    "fees": 12277227
      +    "pk": "031e27beab583e2c94cb3167d128fc1a356c1ae88adfcfaa2334abffa3ae0b4c"
      +    "rewards": 0
      +  }
      +  "03e811dda4f51323ac712cd12299410830d655ddffb104f2c9974d90bf8c583a": {
      +    "fees": 12277227
      +    "pk": "03e811dda4f51323ac712cd12299410830d655ddffb104f2c9974d90bf8c583a"
      +    "rewards": 0
      +  }
      +  "0779ca873bbda77f2850965c8a3a3d40a6ee4ec56af55f0a3f16c7c34c0f298b": {
      +    "fees": 12277227
      +    "pk": "0779ca873bbda77f2850965c8a3a3d40a6ee4ec56af55f0a3f16c7c34c0f298b"
      +    "rewards": 0
      +  }
      +  "07935c642c7409c365258c8488760e96a851cee618aec72eeeb135c9c827f0f9": {
      +    "fees": 12277227
      +    "pk": "07935c642c7409c365258c8488760e96a851cee618aec72eeeb135c9c827f0f9"
      +    "rewards": 0
      +  }
      +  "141b16ac8d5bd150f16b1caa08f689057ca4c4434445e56661831f4e671b7c0a": {
      +    "fees": 12277227
      +    "pk": "141b16ac8d5bd150f16b1caa08f689057ca4c4434445e56661831f4e671b7c0a"
      +    "rewards": 0
      +  }
      +  "19ffdf99dee16e4be2db4b0e000b56ab3a4e10bee9f457d8988f75ff7a79fc00": {
      +    "fees": 12277227
      +    "pk": "19ffdf99dee16e4be2db4b0e000b56ab3a4e10bee9f457d8988f75ff7a79fc00"
      +    "rewards": 0
      +  }
      +  "1af35b29ca515ff5b805a5e3a0ab8c518915b780d5988e76b0672a71b5a3be02": {
      +    "fees": 12277227
      +    "pk": "1af35b29ca515ff5b805a5e3a0ab8c518915b780d5988e76b0672a71b5a3be02"
      +    "rewards": 0
      +  }
      +  "1b5a93c7622c666b0228236a70ee1a31407828b71bfb6daaa29a1509e87d4d3c": {
      +    "fees": 12277227
      +    "pk": "1b5a93c7622c666b0228236a70ee1a31407828b71bfb6daaa29a1509e87d4d3c"
      +    "rewards": 0
      +  }
      +  "1cc68fa0b12521158e09779fd5978ccc0ac26bf99320e00a9549b542dd9ada16": {
      +    "fees": 12277227
      +    "pk": "1cc68fa0b12521158e09779fd5978ccc0ac26bf99320e00a9549b542dd9ada16"
      +    "rewards": 0
      +  }
      +  "1e6ce18addd973ad432f05f16a4c86372eaca054cbdbcaf1169ad6df033f6b85": {
      +    "fees": 12277227
      +    "pk": "1e6ce18addd973ad432f05f16a4c86372eaca054cbdbcaf1169ad6df033f6b85"
      +    "rewards": 0
      +  }
      +  "1e82c7db09da2010e7f5fef24d83bc46238a20ef7ecdf12d9f32e4318a818777": {
      +    "fees": 12277227
      +    "pk": "1e82c7db09da2010e7f5fef24d83bc46238a20ef7ecdf12d9f32e4318a818777"
      +    "rewards": 0
      +  }
      +  "27f43391cca75cbc82d1750307649508d1d318cd015f1f172b97318f17ab954e": {
      +    "fees": 12277227
      +    "pk": "27f43391cca75cbc82d1750307649508d1d318cd015f1f172b97318f17ab954e"
      +    "rewards": 0
      +  }
      +  "2b6f49383af36fd9f1a72d5d2708c8c354add89aaea7edc702c420e2d5fdf22e": {
      +    "fees": 12277227
      +    "pk": "2b6f49383af36fd9f1a72d5d2708c8c354add89aaea7edc702c420e2d5fdf22e"
      +    "rewards": 0
      +  }
      +  "2f9b9a43b915bb8dcea45ea3b8552ebec202eb196a7889c2495d948e15f4a724": {
      +    "fees": 12277227
      +    "pk": "2f9b9a43b915bb8dcea45ea3b8552ebec202eb196a7889c2495d948e15f4a724"
      +    "rewards": 0
      +  }
      +  "31402977c7eaf9e38d18d0689a45d719d615de941f7e80f6db388453b46f4df5": {
      +    "fees": 12277227
      +    "pk": "31402977c7eaf9e38d18d0689a45d719d615de941f7e80f6db388453b46f4df5"
      +    "rewards": 0
      +  }
      +  "3476bba16437ee0e04a29daa34d753139fbcfc14152372d7be5b7c75d51bac6c": {
      +    "fees": 12277227
      +    "pk": "3476bba16437ee0e04a29daa34d753139fbcfc14152372d7be5b7c75d51bac6c"
      +    "rewards": 0
      +  }
      +  "386217d98eee87268a54d2d76ce9e801ac86271284d793154989e37cb31bcd0e": {
      +    "fees": 12277227
      +    "pk": "386217d98eee87268a54d2d76ce9e801ac86271284d793154989e37cb31bcd0e"
      +    "rewards": 0
      +  }
      +  "399a7d14610c4da8800ed929fc6a05133deb8fbac8403dec93226e96fa7590ee": {
      +    "fees": 12277227
      +    "pk": "399a7d14610c4da8800ed929fc6a05133deb8fbac8403dec93226e96fa7590ee"
      +    "rewards": 0
      +  }
      +  "3be2eb47134d5158e5f7d52076b624b76744b3fba8aa50791b46ba21408524c9": {
      +    "fees": 12277227
      +    "pk": "3be2eb47134d5158e5f7d52076b624b76744b3fba8aa50791b46ba21408524c9"
      +    "rewards": 0
      +  }
      +  "3ea481498521e9fb1201b2295d0e9afa826ac6a3ef51de2f00365f915ac7ac06": {
      +    "fees": 12277227
      +    "pk": "3ea481498521e9fb1201b2295d0e9afa826ac6a3ef51de2f00365f915ac7ac06"
      +    "rewards": 0
      +  }
      +  "3ff32442bb6da7d60c1b7752b24e6467813c9b698e0f278d48c43580da972135": {
      +    "fees": 12277227
      +    "pk": "3ff32442bb6da7d60c1b7752b24e6467813c9b698e0f278d48c43580da972135"
      +    "rewards": 0
      +  }
      +  "47b9b07df72d38c19867c6a8c12429e6b8e4d2be48b27cd407da590c7a2af0dc": {
      +    "fees": 12277227
      +    "pk": "47b9b07df72d38c19867c6a8c12429e6b8e4d2be48b27cd407da590c7a2af0dc"
      +    "rewards": 0
      +  }
      +  "47c8b3d6a9e418f0920ef58383260bcd04799db150612d4ff6eb399bcd07f216": {
      +    "fees": 12277227
      +    "pk": "47c8b3d6a9e418f0920ef58383260bcd04799db150612d4ff6eb399bcd07f216"
      +    "rewards": 0
      +  }
      +  "4bde949c19a0803631768148019473929b5f8661e9e48efb8d895efa9dd24aef": {
      +    "fees": 12277227
      +    "pk": "4bde949c19a0803631768148019473929b5f8661e9e48efb8d895efa9dd24aef"
      +    "rewards": 0
      +  }
      +  "4fe5cd087a319956ddc05725651e56486961b7d5733ecd23e26e463bf9253bb5": {
      +    "fees": 12277227
      +    "pk": "4fe5cd087a319956ddc05725651e56486961b7d5733ecd23e26e463bf9253bb5"
      +    "rewards": 0
      +  }
      +  "526931663cbee883ff22369172cba091a5dd5fa1200284fa790d7aeca53d37af": {
      +    "fees": 12277227
      +    "pk": "526931663cbee883ff22369172cba091a5dd5fa1200284fa790d7aeca53d37af"
      +    "rewards": 0
      +  }
      +  "55405aed8c3a1eabe678be3ad4d36043d6ef8e637d213b84ee703d87f6b250ed": {
      +    "fees": 12277227
      +    "pk": "55405aed8c3a1eabe678be3ad4d36043d6ef8e637d213b84ee703d87f6b250ed"
      +    "rewards": 0
      +  }
      +  "5c4af5cb0c1c92df2ed4feeb9751e54e951f9d3f77196511f13e636cf6064e74": {
      +    "fees": 12277227
      +    "pk": "5c4af5cb0c1c92df2ed4feeb9751e54e951f9d3f77196511f13e636cf6064e74"
      +    "rewards": 0
      +  }
      +  "5d28e992b80172f38d3a2f9592cad740fd18d3c2e187745cd5f7badf285ed819": {
      +    "fees": 12277227
      +    "pk": "5d28e992b80172f38d3a2f9592cad740fd18d3c2e187745cd5f7badf285ed819"
      +    "rewards": 0
      +  }
      +  "5f6cc5a8aac752d37c676b0d46a798f7625e37dfa1e96091983274e04ab7ffe2": {
      +    "fees": 12277227
      +    "pk": "5f6cc5a8aac752d37c676b0d46a798f7625e37dfa1e96091983274e04ab7ffe2"
      +    "rewards": 0
      +  }
      +  "6164b0cc68f8de44cde90c78e838b9ee1d6041fa61cf0cfbd834d76bb369a10e": {
      +    "fees": 12277227
      +    "pk": "6164b0cc68f8de44cde90c78e838b9ee1d6041fa61cf0cfbd834d76bb369a10e"
      +    "rewards": 0
      +  }
      +  "62bbb3c41e43df73de2c3f87e6577d095b84cf6deb1b2d6e87612a9156b980f8": {
      +    "fees": 12277227
      +    "pk": "62bbb3c41e43df73de2c3f87e6577d095b84cf6deb1b2d6e87612a9156b980f8"
      +    "rewards": 0
      +  }
      +  "640dfec4541daed209a455577d7ba519ad92b18692edd9ae71d1a02958f47b1b": {
      +    "fees": 12277227
      +    "pk": "640dfec4541daed209a455577d7ba519ad92b18692edd9ae71d1a02958f47b1b"
      +    "rewards": 0
      +  }
      +  "644a971f2c0d0d4b657d050fca27e5f9265e3dfa02a71f7fbf834cc2f2a6a4c8": {
      +    "fees": 12277227
      +    "pk": "644a971f2c0d0d4b657d050fca27e5f9265e3dfa02a71f7fbf834cc2f2a6a4c8"
      +    "rewards": 0
      +  }
      +  "64db2bce729e302f6021047dfd39b6c53caf83b42da4b5b881cb153a3fb31613": {
      +    "fees": 12277227
      +    "pk": "64db2bce729e302f6021047dfd39b6c53caf83b42da4b5b881cb153a3fb31613"
      +    "rewards": 0
      +  }
      +  "67651d29dc8d94bcb1174d5bd602762850a89850503b01a5ffde3b726b43d3d2": {
      +    "fees": 12277227
      +    "pk": "67651d29dc8d94bcb1174d5bd602762850a89850503b01a5ffde3b726b43d3d2"
      +    "rewards": 0
      +  }
      +  "68680ca0bcd4676489976837edeac305c34f652e970386013ef26e67589a2516": {
      +    "fees": 12277227
      +    "pk": "68680ca0bcd4676489976837edeac305c34f652e970386013ef26e67589a2516"
      +    "rewards": 0
      +  }
      +  "6d462852d410e84ca199a34d7ccad443784471f22cf3de37c531ce3b87ebbc41": {
      +    "fees": 12277227
      +    "pk": "6d462852d410e84ca199a34d7ccad443784471f22cf3de37c531ce3b87ebbc41"
      +    "rewards": 0
      +  }
      +  "6e904b2f678eb3b6c3042acb188a607d903d441d61508d047fe36b3c982995c8": {
      +    "fees": 12277227
      +    "pk": "6e904b2f678eb3b6c3042acb188a607d903d441d61508d047fe36b3c982995c8"
      +    "rewards": 0
      +  }
      +  "6f04988de7e63537c8f14e84b0eb51e0ea9c5da8b4b9256243b3e40b1aeccb76": {
      +    "fees": 12277227
      +    "pk": "6f04988de7e63537c8f14e84b0eb51e0ea9c5da8b4b9256243b3e40b1aeccb76"
      +    "rewards": 0
      +  }
      +  "6fb2e0882cd9d895e1e441b9f9be7f98e877aa0a16ae230ee5caceb7a1b896ae": {
      +    "fees": 12277227
      +    "pk": "6fb2e0882cd9d895e1e441b9f9be7f98e877aa0a16ae230ee5caceb7a1b896ae"
      +    "rewards": 0
      +  }
      +  "73fec19d4bfe361c0680a7cfd24b3f744a1c1b29d932c4d89ce6157679f8af7d": {
      +    "fees": 12277227
      +    "pk": "73fec19d4bfe361c0680a7cfd24b3f744a1c1b29d932c4d89ce6157679f8af7d"
      +    "rewards": 0
      +  }
      +  "74583aba9c0b92e4f08c8c75e6df341c255ca007971195ff64d6f909dc4b7177": {
      +    "fees": 12277227
      +    "pk": "74583aba9c0b92e4f08c8c75e6df341c255ca007971195ff64d6f909dc4b7177"
      +    "rewards": 0
      +  }
      +  "76c9494237e608d43fd6fb0114106a7517f5503cf79d7482db58a02304339b6c": {
      +    "fees": 12277227
      +    "pk": "76c9494237e608d43fd6fb0114106a7517f5503cf79d7482db58a02304339b6c"
      +    "rewards": 0
      +  }
      +  "82174ee408161186e650427032f4cfb2496f429b4157da78888cbcea39c387fc": {
      +    "fees": 12277227
      +    "pk": "82174ee408161186e650427032f4cfb2496f429b4157da78888cbcea39c387fc"
      +    "rewards": 0
      +  }
      +  "85b07e51ffe528f272b7eb734d0496158f2b0f890155ebe59ba2989a8ccc9a49": {
      +    "fees": 12277227
      +    "pk": "85b07e51ffe528f272b7eb734d0496158f2b0f890155ebe59ba2989a8ccc9a49"
      +    "rewards": 0
      +  }
      +  "86499879448d1b0215d59cbf078836e3d7d9d2782d56a2274a568761bff36f19": {
      +    "fees": 12277227
      +    "pk": "86499879448d1b0215d59cbf078836e3d7d9d2782d56a2274a568761bff36f19"
      +    "rewards": 0
      +  }
      +  "8a0bcba8e909036b7a0fdb244f049d847b117d871d203ef7cc4c3917c94fd5fd": {
      +    "fees": 12277227
      +    "pk": "8a0bcba8e909036b7a0fdb244f049d847b117d871d203ef7cc4c3917c94fd5fd"
      +    "rewards": 0
      +  }
      +  "904c294899819cce0283d8d351cb10febfa0e9f0acd90a820ec8eb90a7084c37": {
      +    "fees": 12277227
      +    "pk": "904c294899819cce0283d8d351cb10febfa0e9f0acd90a820ec8eb90a7084c37"
      +    "rewards": 0
      +  }
      +  "910da2a8e20f25ccbcb029fdcafd369b43d75e5bc4dc6d92352c29404acc350f": {
      +    "fees": 12277227
      +    "pk": "910da2a8e20f25ccbcb029fdcafd369b43d75e5bc4dc6d92352c29404acc350f"
      +    "rewards": 0
      +  }
      +  "94b163c5a5ad346db1c84edaff51604164476cf78b8834b6b610dd03bd6b65d9": {
      +    "fees": 12277227
      +    "pk": "94b163c5a5ad346db1c84edaff51604164476cf78b8834b6b610dd03bd6b65d9"
      +    "rewards": 0
      +  }
      +  "9503d36c0810f9ac1a9d7d45bf778387a2baab151a45d77ac1289fbe29abb18f": {
      +    "fees": 12277227
      +    "pk": "9503d36c0810f9ac1a9d7d45bf778387a2baab151a45d77ac1289fbe29abb18f"
      +    "rewards": 0
      +  }
      +  "95ea7eb026e250741be85e3593166ef0c4cb3a6eb9114dba8f0974987f10403f": {
      +    "fees": 12277227
      +    "pk": "95ea7eb026e250741be85e3593166ef0c4cb3a6eb9114dba8f0974987f10403f"
      +    "rewards": 0
      +  }
      +  "96c16a6251e1b9a8c918d5821a5aa8dfb9385607258338297221c5a226eca5c6": {
      +    "fees": 12277227
      +    "pk": "96c16a6251e1b9a8c918d5821a5aa8dfb9385607258338297221c5a226eca5c6"
      +    "rewards": 0
      +  }
      +  "9986cedd4b5a28e4c81d9b4bff0461dddaa25099df00b8632fe99e88df28ce73": {
      +    "fees": 12277227
      +    "pk": "9986cedd4b5a28e4c81d9b4bff0461dddaa25099df00b8632fe99e88df28ce73"
      +    "rewards": 0
      +  }
      +  "9a0f19e60581003b70291cf4a874e8217b04871e676b2c53c85a18ab95c2683b": {
      +    "fees": 12277227
      +    "pk": "9a0f19e60581003b70291cf4a874e8217b04871e676b2c53c85a18ab95c2683b"
      +    "rewards": 0
      +  }
      +  "9a7452495138cf7cf5a1564c3ef16b186dd8ab4f96423f160e22a3aec6eb614f": {
      +    "fees": 12277227
      +    "pk": "9a7452495138cf7cf5a1564c3ef16b186dd8ab4f96423f160e22a3aec6eb614f"
      +    "rewards": 0
      +  }
      +  "9c16751dbe57f4dff7b3fb8911a62c0cb2bdee6240e3f3fefe76832788cb14c6": {
      +    "fees": 12277227
      +    "pk": "9c16751dbe57f4dff7b3fb8911a62c0cb2bdee6240e3f3fefe76832788cb14c6"
      +    "rewards": 0
      +  }
      +  "9d3058175acab969f41ad9b86f7a2926c74258670fe56b37c429c01fca9f2f0f": {
      +    "fees": 12277227
      +    "pk": "9d3058175acab969f41ad9b86f7a2926c74258670fe56b37c429c01fca9f2f0f"
      +    "rewards": 0
      +  }
      +  "9f2fcc688518324273da230afff9756312bf23592174896fab669c2d78b1533c": {
      +    "fees": 12277227
      +    "pk": "9f2fcc688518324273da230afff9756312bf23592174896fab669c2d78b1533c"
      +    "rewards": 0
      +  }
      +  "a10ed9c59dac2c4b8264dc34f2d318719fb5f20ecdd8d6be2d7abfe32294f20d": {
      +    "fees": 12277227
      +    "pk": "a10ed9c59dac2c4b8264dc34f2d318719fb5f20ecdd8d6be2d7abfe32294f20d"
      +    "rewards": 0
      +  }
      +  "a10f963752b3a44702dfa48b429ac742bea94d97849b1180a36750df3a783621": {
      +    "fees": 12277227
      +    "pk": "a10f963752b3a44702dfa48b429ac742bea94d97849b1180a36750df3a783621"
      +    "rewards": 0
      +  }
      +  "a50a55d4476bb118ba5121a07b51c185a8fe0a92b65840143b006b9820124df4": {
      +    "fees": 12277227
      +    "pk": "a50a55d4476bb118ba5121a07b51c185a8fe0a92b65840143b006b9820124df4"
      +    "rewards": 0
      +  }
      +  "a796e9c0516a40ccd0eee7a32fdc2dc297fee40a9c76fef9c1bb0cf41ae69750": {
      +    "fees": 12277227
      +    "pk": "a796e9c0516a40ccd0eee7a32fdc2dc297fee40a9c76fef9c1bb0cf41ae69750"
      +    "rewards": 0
      +  }
      +  "aa33af13b440746b4f24312cba5fa910eb077ce6b16b84ebb482cb7720b5c686": {
      +    "fees": 12277227
      +    "pk": "aa33af13b440746b4f24312cba5fa910eb077ce6b16b84ebb482cb7720b5c686"
      +    "rewards": 0
      +  }
      +  "addb0e15a44b0fdc6ff291be28d8c98f5551d0cd9218d749e30ddb87c6e31ca9": {
      +    "fees": 12277227
      +    "pk": "addb0e15a44b0fdc6ff291be28d8c98f5551d0cd9218d749e30ddb87c6e31ca9"
      +    "rewards": 0
      +  }
      +  "b00269bd169f0f89bd2f278788616521dd1539868ced5a63b652208a04ee1556": {
      +    "fees": 12277227
      +    "pk": "b00269bd169f0f89bd2f278788616521dd1539868ced5a63b652208a04ee1556"
      +    "rewards": 0
      +  }
      +  "b137de324fcc79dd1a21ae39a2ee8eed05e76b86d8e89d378f8bb766afb8719f": {
      +    "fees": 12277227
      +    "pk": "b137de324fcc79dd1a21ae39a2ee8eed05e76b86d8e89d378f8bb766afb8719f"
      +    "rewards": 0
      +  }
      +  "b5341e839b25c4cc2aaf421704c0fb6ba987d537678e23e45d3ca32454a2908c": {
      +    "fees": 12277227
      +    "pk": "b5341e839b25c4cc2aaf421704c0fb6ba987d537678e23e45d3ca32454a2908c"
      +    "rewards": 0
      +  }
      +  "b6ac700bf890b887e218dbd55b8f6b091dfc5a684d0fd7a6f69db7dc0313b51b": {
      +    "fees": 12277227
      +    "pk": "b6ac700bf890b887e218dbd55b8f6b091dfc5a684d0fd7a6f69db7dc0313b51b"
      +    "rewards": 0
      +  }
      +  "ba2ea5e324eeb42fa6f4d1132a1d79911721e8507033bb0abd49715f531877b4": {
      +    "fees": 12277227
      +    "pk": "ba2ea5e324eeb42fa6f4d1132a1d79911721e8507033bb0abd49715f531877b4"
      +    "rewards": 0
      +  }
      +  "ba7acc3bcbd47dbf13d744e57f696341c260ce2ea8f332919f18cb543b1f3fc7": {
      +    "fees": 12277227
      +    "pk": "ba7acc3bcbd47dbf13d744e57f696341c260ce2ea8f332919f18cb543b1f3fc7"
      +    "rewards": 0
      +  }
      +  "bf9f5cfc548d29983cc0dfa5c4ec47c66c31df0f87aa669869678996902ab47f": {
      +    "fees": 12277227
      +    "pk": "bf9f5cfc548d29983cc0dfa5c4ec47c66c31df0f87aa669869678996902ab47f"
      +    "rewards": 0
      +  }
      +  "c3d1bc76dea367512df3832c437c7b2c95508e140f655425a733090da86fb82d": {
      +    "fees": 12277227
      +    "pk": "c3d1bc76dea367512df3832c437c7b2c95508e140f655425a733090da86fb82d"
      +    "rewards": 0
      +  }
      +  "c4dfedeb4f639f749e498a2307f1545ddd6bda62e5503ac1832b122c4a5aedf9": {
      +    "fees": 12277227
      +    "pk": "c4dfedeb4f639f749e498a2307f1545ddd6bda62e5503ac1832b122c4a5aedf9"
      +    "rewards": 0
      +  }
      +  "c61d0822bbdbfe2a0b5503daff0ce8441c623115c94c0cfcf047a51f8b7160d3": {
      +    "fees": 12277227
      +    "pk": "c61d0822bbdbfe2a0b5503daff0ce8441c623115c94c0cfcf047a51f8b7160d3"
      +    "rewards": 0
      +  }
      +  "cdd68a321ea737e82bce23d2208040f79471d36f2e6f84c74ea36ab26245e522": {
      +    "fees": 12277227
      +    "pk": "cdd68a321ea737e82bce23d2208040f79471d36f2e6f84c74ea36ab26245e522"
      +    "rewards": 0
      +  }
      +  "cf8a3bf23d1936a34facc4ff63d86d21cc2e1ac17e0010035dc3ef7ae85010dc": {
      +    "fees": 12277227
      +    "pk": "cf8a3bf23d1936a34facc4ff63d86d21cc2e1ac17e0010035dc3ef7ae85010dc"
      +    "rewards": 0
      +  }
      +  "d1c3a2cb254554971db289b917a665b5c547617d6fd20c2d6051bc5dfc805b34": {
      +    "fees": 12277227
      +    "pk": "d1c3a2cb254554971db289b917a665b5c547617d6fd20c2d6051bc5dfc805b34"
      +    "rewards": 0
      +  }
      +  "d3e3c8348bca51461eabfc382f8a01e8e284db54104ad37ec0695d48ae5531ac": {
      +    "fees": 12277227
      +    "pk": "d3e3c8348bca51461eabfc382f8a01e8e284db54104ad37ec0695d48ae5531ac"
      +    "rewards": 0
      +  }
      +  "d8daea40fd098d4d546aa76b8e006ce4368c052ffe2c26b6eb843e925d54a408": {
      +    "fees": 12277227
      +    "pk": "d8daea40fd098d4d546aa76b8e006ce4368c052ffe2c26b6eb843e925d54a408"
      +    "rewards": 0
      +  }
      +  "da673805f349faf9ca1db167cb941b27f4517a36d23b3c21da4159cff0045fbe": {
      +    "fees": 12277227
      +    "pk": "da673805f349faf9ca1db167cb941b27f4517a36d23b3c21da4159cff0045fbe"
      +    "rewards": 0
      +  }
      +  "db821a4f828db977c6a8d186cc4a44280a6ef6f54ac18ec9eb32f78735f38683": {
      +    "fees": 12277227
      +    "pk": "db821a4f828db977c6a8d186cc4a44280a6ef6f54ac18ec9eb32f78735f38683"
      +    "rewards": 0
      +  }
      +  "e13a0267444e026fe755ec128858bf3c519864631e0e4c474ba33f2470a18b83": {
      +    "fees": 12277227
      +    "pk": "e13a0267444e026fe755ec128858bf3c519864631e0e4c474ba33f2470a18b83"
      +    "rewards": 0
      +  }
      +  "e42bfabc4a61f02131760af5f2fa0311007932a819a508da25f2ce6af2468156": {
      +    "fees": 12277227
      +    "pk": "e42bfabc4a61f02131760af5f2fa0311007932a819a508da25f2ce6af2468156"
      +    "rewards": 0
      +  }
      +  "e6d075e3e396673c853210f74f8fe6db5e814c304bb9cd7f362018881a21f76c": {
      +    "fees": 12277227
      +    "pk": "e6d075e3e396673c853210f74f8fe6db5e814c304bb9cd7f362018881a21f76c"
      +    "rewards": 0
      +  }
      +  "e818ac2e8e9ffacd2d49f0f2f6739e16711644194d10bb1a8e9e434603125fa1": {
      +    "fees": 12277227
      +    "pk": "e818ac2e8e9ffacd2d49f0f2f6739e16711644194d10bb1a8e9e434603125fa1"
      +    "rewards": 0
      +  }
      +  "eabfe7093ef2394deb1b84287f2ceb1b55fe638edc3358a28fc74f64b3498094": {
      +    "fees": 12277227
      +    "pk": "eabfe7093ef2394deb1b84287f2ceb1b55fe638edc3358a28fc74f64b3498094"
      +    "rewards": 0
      +  }
      +  "edbb9828fbe62da2a59afbc8623e8ebc5ed2f9b7f77a0cd1cdcf55edea30521c": {
      +    "fees": 12277227
      +    "pk": "edbb9828fbe62da2a59afbc8623e8ebc5ed2f9b7f77a0cd1cdcf55edea30521c"
      +    "rewards": 0
      +  }
      +  "f25af3c59ac7f5155c7a9f36762bd941b9dc9c5c051a1bc2d4e34ed773dd04a3": {
      +    "fees": 12277227
      +    "pk": "f25af3c59ac7f5155c7a9f36762bd941b9dc9c5c051a1bc2d4e34ed773dd04a3"
      +    "rewards": 0
      +  }
      +  "f33f93aa1f3ddcfd4e42d3206ddaab966f7f1b6672e5096d6da6adefd38edc67": {
      +    "fees": 12277227
      +    "pk": "f33f93aa1f3ddcfd4e42d3206ddaab966f7f1b6672e5096d6da6adefd38edc67"
      +    "rewards": 0
      +  }
      +  "f62062b7590d46f382fb8c37a26ab0a1bd512951777aedcaa96822230727d3a1": {
      +    "fees": 12277227
      +    "pk": "f62062b7590d46f382fb8c37a26ab0a1bd512951777aedcaa96822230727d3a1"
      +    "rewards": 0
      +  }
      +  "f7b9751d59dd6be6029aa36a81a3f6436e2970cf4348845ab6254678fb946c18": {
      +    "fees": 12277227
      +    "pk": "f7b9751d59dd6be6029aa36a81a3f6436e2970cf4348845ab6254678fb946c18"
      +    "rewards": 0
      +  }
      +  "f827f60366fae9f9ed65384979de780f4a18c6dbfbefb1c7d100957dde51a06d": {
      +    "fees": 12277227
      +    "pk": "f827f60366fae9f9ed65384979de780f4a18c6dbfbefb1c7d100957dde51a06d"
      +    "rewards": 0
      +  }
      +  "f9f6ff873c10c24eba834be28a56415a49c9c67b7c0ee9f106da827847168986": {
      +    "fees": 12277227
      +    "pk": "f9f6ff873c10c24eba834be28a56415a49c9c67b7c0ee9f106da827847168986"
      +    "rewards": 0
      +  }
      +  "fab7b58be4c1e9542c342023b52e9d359ea89a3af34440bdb97318273e8555f0": {
      +    "fees": 12277300
      +    "pk": "fab7b58be4c1e9542c342023b52e9d359ea89a3af34440bdb97318273e8555f0"
      +    "rewards": 0
      +  }
      +  "fc8672466cc16688b5e239a784cd0e4c0acf214af039d9b2bf7a006da4043883": {
      +    "fees": 12277227
      +    "pk": "fc8672466cc16688b5e239a784cd0e4c0acf214af039d9b2bf7a006da4043883"
      +    "rewards": 0
      +  }
      +}
      
    at Proxy.assertEqual (node_modules/chai/lib/chai/core/assertions.js:951:19)
    at Proxy.methodWrapper (node_modules/chai/lib/chai/utils/addMethod.js:57:25)
    at test/unit/sql/rounds.js:638:29
    at tryCatcher (node_modules/bluebird/js/release/util.js:16:23)
    at Holder$3._callFunction (<anonymous>:14:44)
    at Holder$3.checkFulfillment (<anonymous>:29:30)
    at Promise.eval (<anonymous>:6:20)
    at Promise._settlePromise (node_modules/bluebird/js/release/promise.js:566:21)
    at Promise._settlePromise0 (node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (node_modules/bluebird/js/release/promise.js:693:18)
    at Async._drainQueue (node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.Async.drainQueues (node_modules/bluebird/js/release/async.js:17:14)
  

  4) Rounds-related SQL triggers
       round
         should generate a different delegate list than one generated at the beginning of round 1:

      AssertionError: expected [ Array(101) ] to not deeply equal [ Array(101) ]
      + expected - actual


    at Proxy.assertEqual (node_modules/chai/lib/chai/core/assertions.js:951:19)
    at Proxy.methodWrapper (node_modules/chai/lib/chai/utils/addMethod.js:57:25)
    at Context.<anonymous> (test/unit/sql/rounds.js:660:41)
  

  5) Rounds-related SQL triggers
       round
         Round rollback (table delegates) when forger of last block of round is unvoted
           last block height should be at height 99 after deleting one more block:

      AssertionError: expected 98 to equal 99
      + expected - actual

      -98
      +99
      
    at Context.<anonymous> (test/unit/sql/rounds.js:713:34)
  

  6) Rounds-related SQL triggers
       round
         Round rollback (table delegates) when forger of last block of round is unvoted
           after finishing round, delegates list should be different than one generated at the beginning of round 1:

      AssertionError: expected [ Array(101) ] to not deeply equal [ Array(101) ]
      + expected - actual


    at Proxy.assertEqual (node_modules/chai/lib/chai/core/assertions.js:951:19)
    at Proxy.methodWrapper (node_modules/chai/lib/chai/utils/addMethod.js:57:25)
    at test/unit/sql/rounds.js:751:44
    at tryCatcher (node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (node_modules/bluebird/js/release/promise.js:693:18)
    at Async._drainQueue (node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.Async.drainQueues (node_modules/bluebird/js/release/async.js:17:14)
  

  7) Rounds-related SQL triggers
       round
         Round rollback (table delegates) when forger of last block of round is unvoted
           forger of last block of previous round should have voters_balance and voters_cnt 0:

      AssertionError: expected 0 to equal 1
      + expected - actual

      -0
      +1
      
    at test/unit/sql/rounds.js:758:65
  

  8) Rounds-related SQL triggers
       round
         Round rollback (table delegates) when forger of last block of round is replaced in last block of round
           last block height should be at height 100:

      AssertionError: expected 99 to equal 100
      + expected - actual

      -99
      +100
      
    at Context.<anonymous> (test/unit/sql/rounds.js:814:34)
  

  9) Rounds-related SQL triggers
       round
         Round rollback (table delegates) when forger of last block of round is replaced in last block of round
           delegates list should be different than one generated at the beginning of round 1:

      AssertionError: expected [ Array(101) ] to not deeply equal [ Array(101) ]
      + expected - actual


    at Proxy.assertEqual (node_modules/chai/lib/chai/core/assertions.js:951:19)
    at Proxy.methodWrapper (node_modules/chai/lib/chai/utils/addMethod.js:57:25)
    at Context.<anonymous> (test/unit/sql/rounds.js:839:42)
  

  10) Rounds-related SQL triggers
       round
         Round rollback (table delegates) when forger of last block of round is replaced in last block of round
           unvoted delegate should not be on list:
     AssertionError: expected [ Array(101) ] to not include 'fab7b58be4c1e9542c342023b52e9d359ea89a3af34440bdb97318273e8555f0'
    at Context.<anonymous> (test/unit/sql/rounds.js:844:37)
  

  11) Rounds-related SQL triggers
       round
         Round rollback (table delegates) when forger of last block of round is replaced in last block of round
           delegate who replaced unvoted one should be on list:
     AssertionError: expected [ Array(101) ] to include '5351ad0b4b60c0294278f9691ba3c5356452d0ee8b2a3c23ad8531daef1ab989'
    at Context.<anonymous> (test/unit/sql/rounds.js:849:33)
  

  12) Rounds-related SQL triggers
       round
         Round rollback (table delegates) when forger of last block of round is replaced in last block of round
           forger of last block of previous round should have voters_balance and voters_cnt 0:

      AssertionError: expected 0 to equal 1
      + expected - actual

      -0
      +1
      
    at test/unit/sql/rounds.js:855:65
  

  13) Rounds-related SQL triggers
       round
         Round rollback (table delegates) when forger of last block of round is replaced in last block of round
           delegate who replaced last block forger should have proper votes:
     AssertionError: expected 0 to be above 0
    at test/unit/sql/rounds.js:866:45
```

### Steps to reproduce

Run unit and functional test-suite on Jenkins. Individual test-suite works locally.

---

Related to: https://github.com/LiskHQ/lisk/issues/969

--- a/test/common/application.js
+++ b/test/common/application.js
@@ -276,6 +276,12 @@ function __init (initScope, done) {
 		}, function (err, scope) {
 			scope.rewiredModules = rewiredModules;
 			currentAppScope = scope;
+			test.debug('initApplication: Rewired modules available');
+
+			// Overwrite syncing function to prevent interfere with tests
+			scope.modules.loader.syncing = function () {
+				return false;
+			};
 
 			// If bus is overridden, then we just return the scope, without waiting for genesisBlock
 			if (!initScope.waitForGenesisBlock || initScope.bus) {

--- a/test/unit/sql/rounds.js
+++ b/test/unit/sql/rounds.js
@@ -154,7 +154,7 @@ describe('Rounds-related SQL triggers', function () {
 			sandbox: {
 				name: 'lisk_test_sql_rounds'
 			},
-			waitForGenesisBlock: false
+			waitForGenesisBlock: true
 		}, function (err, scope) {
 			// Force rewards start at 150-th block
 			library = scope;
@@ -196,12 +196,6 @@ describe('Rounds-related SQL triggers', function () {
 			}, []);
 		});
 
-		it('should not populate mem_accounts', function () {
-			return getMemAccounts().then(function (accounts) {
-				expect(Object.keys(accounts).length).to.equal(0);
-			});
-		});
-
 		it('should load genesis block with transactions into database (native)', function (done) {
 			library.db.query('SELECT * FROM full_blocks_list WHERE b_height = 1').then(function (rows) {
 				genesisBlock = library.modules.blocks.utils.readDbRows(rows)[0];
@@ -235,10 +229,7 @@ describe('Rounds-related SQL triggers', function () {
 		});
 
 		it('should apply genesis block transactions to mem_accounts (native)', function () {
-			// Wait 20 seconds for proper initialisation
-			return Promise.delay(20000).then(function () {
-				return getMemAccounts();
-			}).then(function (accounts) {
+			return getMemAccounts().then(function (accounts) {
 				// Number of returned accounts should be equal to number of unique accounts in genesis block
 				expect(Object.keys(accounts).length).to.equal(genesisAccounts.length);
 
@@ -553,13 +544,6 @@ describe('Rounds-related SQL triggers', function () {
 			});
 			return delegates_state;
 		}
-
-		before(function () {
-			return Promise.delay(1000).then(function () {
-				// Set delegates module as loaded to allow manual forging
-				library.rewiredModules.delegates.__set__('__private.loaded', true);
-			});
-		});
 
 		it('should load all secrets of 101 delegates and set modules.delegates.__private.keypairs (native)', function (done) {
 			var loadDelegates = library.rewiredModules.delegates.__get__('__private.loadDelegates');

